brooklyn-docs/guide/ops/externalized-configuration.md

```diff
@@ -122,6 +122,23 @@ brooklyn.location.jclouds.aws-ec2.credential=$brooklyn:external("mysupplier", "a
 {% endhighlight %}
 
 
+## Referring to External Configuration in catalog items
+
+The same blueprint language DSL can be used within YAML catalog items. For example:
+
+    brooklyn.catalog:
+      id: com.example.myblueprint
+      version: 1.2.3
+      brooklyn.libraries:
+      - >
+        $brooklyn:formatString("https://%s:%s@repo.example.com/libs/myblueprint-1.2.3.jar", 
+        external("mysuppier", "username"), external("mysupplier", "password"))
+      item:
+        type: com.example.MyBlueprint
+
+Note the `>` in the example above is used to split across multiple lines.
+
+
```

Comment:
Much better example. I didn't realise that the `$brooklyn` prefix can be dropped inside the scope of the function arguments; does this definitely parse? ;o)